packages/search/src/ago/helpers/format/format-item.ts

```diff
@@ -48,7 +48,8 @@ function formatItemAttributes(item: IItem) {
     // computed or null attributes so items & datasets look the same
     name: item.title,
     searchDescription: item.description,
-    hubType: hubType || "Other"
+    hubType: hubType || "Other",
+    collection: getTypeCategories({}, item)
```

Comment:
should we consider that `collection` could be added to item attributes in the future?

Reply:
Good question - I don't think collection will be part of AGO because it is a hub specific terminology. May be @dmfenton can weigh in on this.